.github/workflows/runTests.yml

```diff
@@ -18,4 +18,4 @@ jobs:
     - name: Build with Gradle
       uses: gradle/gradle-build-action@v1.5.1
       with:
-        arguments: testDebugUnitTest lintRelease checkstyle lintKotlin --stacktrace
+        arguments: testDebugUnitTest lintRelease checkstyle lintKotlin --debug
```

Comment:
I see a couple of suspicious notes in the logs. A bunch of these:
```
2021-09-22T01:15:40.9881485Z 2021-09-22T01:15:05.910+0000 [DEBUG] [org.gradle.execution.plan.DefaultExecutionPlan] Cannot acquire worker lease lock for node :magellan-test:lintKotlinTestRelease
```

And many of these:
```
2021-09-22T01:15:40.9907377Z 2021-09-22T01:15:04.387+0000 [DEBUG] [org.gradle.internal.component.model.LoggingAttributeMatchingExplanationBuilder] Candidate androidx.test:core-ktx:1.4.0 configuration runtime doesn't match attributes {artifactType=android-res, com.android.build.api.attributes.BuildTypeAttr=debug, org.gradle.usage=java-runtime, org.jetbrains.kotlin.platform.type=androidJvm}
```

Reply:
Look into using build cache:
```
2021-09-22T01:15:41.4725952Z 2021-09-22T01:15:04.582+0000 [INFO] [org.gradle.internal.execution.steps.ResolveCachingStateStep] Caching disabled for task ':magellan-library:mergeDebugUnitTestResources' because:
2021-09-22T01:15:41.4727880Z   Build cache is disabled
```

Reply:
More importantly, towards the end I see:
```
2021-09-22T01:15:45.9043853Z 2021-09-22T01:15:40.246+0000 [DEBUG] [TestEventLogger] 
2021-09-22T01:15:45.9045816Z 2021-09-22T01:15:40.246+0000 [DEBUG] [TestEventLogger] com.wealthfront.magellan.transitions.DefaultTransitionTest > animateGoBack STANDARD_ERROR
2021-09-22T01:15:45.9049079Z 2021-09-22T01:15:40.246+0000 [DEBUG] [TestEventLogger]     System.logW: A resource was acquired at attached stack trace but never released. See java.io.Closeable for information on avoiding resource leaks.java.lang.Throwable: Explicit termination method 'dispose' not called
2021-09-22T01:15:45.9052635Z 2021-09-22T01:15:40.252+0000 [DEBUG] [TestEventLogger]     System.logW: A resource was acquired at attached stack trace but never released. See java.io.Closeable for information on avoiding resource leaks.java.lang.Throwable: Explicit termination method 'dispose' not called
2021-09-22T01:15:45.9066252Z 2021-09-22T01:15:40.254+0000 [DEBUG] [TestEventLogger]     System.logW: A resource was acquired at attached stack trace but never released. See java.io.Closeable for information on avoiding resource leaks.java.lang.Throwable: Explicit termination method 'dispose' not called
```
Followed by many, many:
```
2021-09-22T01:15:45.9074238Z 2021-09-22T01:15:42.475+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
2021-09-22T01:15:45.9076791Z 2021-09-22T01:15:42.475+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
2021-09-22T01:15:45.9078801Z 2021-09-22T01:15:42.475+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
```

Reply:
I notice that sometimes, the animations tests hang locally as well.  Oh wow, is this a freaking LooperMode issue...